client/src/shared/services/storageFactory.js

```diff
@@ -0,0 +1,32 @@
+(function() {
+  'use strict';
+angular
+  .module('app')
+  .factory('storageFactory',['$window', function($window){
+    return {
+      setSession: function( key, value ){
```

Comment:
no space required for parentheses `(key, value)`
Only add space for curly braces `{ ctrl.something }`